src/shared/components/setup-locker/setup-locker-steps/set-passphrase/SetPassphrase.js

```diff
@@ -0,0 +1,253 @@
+import React, { Component } from 'react';
```

Comment:
There's a lot of things going on here. Can we discuss this personally or in a call?